test/action_controller/namespace_lookup_test.rb

```diff
@@ -125,7 +125,7 @@ def namespace_set_by_request_headers
       tests Api::V3::LookupTestController
 
       setup do
-        @test_namespace = self.class.parent
+        @test_namespace = self.class.module_parent
```

Comment:
but to make tests pass for previous rails version you also have to use if Module.method_defined?(:module_parent)

Reply:
good catch - just pushed an update that fixes this